doc_admin/AdminManual.adoc

```diff
@@ -225,16 +228,17 @@ Open a terminal and navigate to the sql/ folder e.g.):
 
 [source,shell]
 ----
-C:\WINDOWS\system32> cd D:\OpenHospital_1_11_0\sql
+C:\WINDOWS> cd D:\OpenHospital-v1.11.0\sql
```

Comment:
small nit-pick; typically the `cd` command in a plain DOS window cannot change both the drive and the directory; it must be done in two different commands.

Reply:
Right, fixed !